common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -99,7 +99,9 @@ public static void generateMaterialBlocks() {
                         .setData(ProviderType.BLOCKSTATE, NonNullBiConsumer.noop())
                         .setData(ProviderType.LANG, NonNullBiConsumer.noop())
                         .setData(ProviderType.LOOT, NonNullBiConsumer.noop())
-                        //.tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.BEACON_BASE_BLOCKS)
+                        // TODO: Add #forge:storage_blocks tag
                         .color(() -> () -> MaterialBlock::tintedColor)
```

Comment:
I'm not sure why it was commented before. Probably he has some ideas for tags @screret 
smth like adding tag for kjs's materials I think?

Reply:
We are injecting those tags runtime.
In TagLoaderMixin

Reply:
so we do not need it anymore?

Reply:
Correct

Reply:
What would you like removed from this section?